source/blog/2024-11-05-hanami-220.html.markdown

```diff
@@ -0,0 +1,244 @@
+---
+title: "Hanami 2.2.0: Persistence pays off"
```

Comment:
```suggestion
title: "Hanami 2.2: Persistence pays off"
```
I think it's cleaner and looks better, plus it matches the format for 2.1 and 2.0 releases.

I really like the name "Persistence pays off". Is there some way we could add a reference to "Operation" too? I was thinking "Operation Persistence is complete" but [that's already the name of something tragic](https://en.wikipedia.org/wiki/Operation_Persistence).

Reply:
Ah yes, thank you! Have updated the title.

In terms of the tag line, I'm glad you like "persistence pays off" :) This was a name I gave to solnic for a talk he did at RubyConf AU back in 2017, but I like it a lot and I figured this was the perfect time to bring it back.

Persistence is clearly the hero of this release, so I think it's fine for that to be the main focus of the tag line. We do make it clear in other places that operations are the next biggest thing :)